src/IntegratedExperts/BehatPhpServer/PhpServerContext.php

```diff
@@ -0,0 +1,281 @@
+<?php
+
+/**
+ * This file is part of the IntegratedExperts Behat tests.
+ *
+ * @file Context PhpServer for Behat.
+ */
+
+namespace IntegratedExperts\BehatPhpServer;
+
+use Behat\Behat\Context\Context;
+use Behat\MinkExtension\Context\MinkContext;
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Symfony\Component\Console\Exception\RuntimeException;
+
+/**
+ * Class PhpServerTrait.
+ */
+class PhpServerContext extends MinkContext implements Context
+{
+
+    /**
+     * Docroot directory.
+     *
+     * @var string
+     */
+    protected $docroot;
+
+    /**
+     * Server hostname.
+     *
+     * @var string
+     */
+    protected $host;
+
+    /**
+     * Server port.
+     *
+     * @var string
+     */
+    protected $port;
+
+    /**
+     * Server process id.
+     *
+     * @var string
+     */
+    protected $pid;
+
+
+    /**
+     * PhpServerTrait constructor.
+     *
+     * @param array $parameters Settings for server.
+     */
+    public function __construct($parameters = [])
+    {
+        $this->docroot = isset($parameters['docroot']) ? $parameters['docroot'] : __DIR__.'/fixtures';
+        $this->host    = isset($parameters['host']) ? $parameters['host'] : 'localhost';
+        $this->port    = isset($parameters['port']) ? $parameters['port'] : '8888';
+
+    }//end __construct()
+
+
+    /**
+     * Start server before each scenario.
+     *
+     * @param BeforeScenarioScope $scope Scenario scope.
+     *
+     * @beforeScenario @phpserver
+     */
+    public function beforeScenarioStartPhpServer(BeforeScenarioScope $scope)
+    {
+        if ($scope->getScenario()->hasTag('phpserver')) {
+            $this->phpServerStart();
+        }
+
+    }//end beforeScenarioStartPhpServer()
+
+
+    /**
+     * Stop server after each scenario.
+     *
+     * @param AfterScenarioScope $scope Scenario scope.
+     *
+     * @afterScenario @phpserver
+     */
+    public function afterScenarioStopPhpServer(AfterScenarioScope $scope)
+    {
+        if ($scope->getScenario()->hasTag('phpserver')) {
+            $this->phpServerStop();
+        }
+
+    }//end afterScenarioStopPhpServer()
+
+
+    /**
+     * Start a server.
+     *
+     * @throws RuntimeException
+     *   If unable to start a server.
+     *
+     * @return int
+     *   PID as number.
+     */
+    protected function phpServerStart()
```

Comment:
please update all method names to not have `phpServer` etc. This was originally done in trait and was providing namespacing for this methods. Since this is own class, it is no longer required.